src/bin/vip-wp.js

```diff
@@ -72,6 +112,66 @@ commandWrapper( {
 
 		let result;
 		let rl;
+		let subShellRl;
+
+		if ( isSubShell ) {
+			console.log( `Entering WP-CLI subshell mode for ${ formatEnvironment( envName ) } on ${ appName } (${ appId })` );
+
+			subShellRl = readline.createInterface( {
+				input: process.stdin,
+				output: process.stdout,
+				terminal: true,
+				prompt: `${ appName }.${ envName }> `,
```

Comment:
To ensure consistency, can we reuse `getEnvIdentifier()` for this?

https://github.com/Automattic/vip/blob/45d34473fe3fafe989fb673ff90a1cb81d06519a/src/lib/cli/command.js#L407

Reply:
Yay, thanks for the suggestions. Fixed here https://github.com/Automattic/vip/pull/280/commits/401fb460c1b47787301d1cb29f78d8a44194391a